test/wsk-preemption.bats

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env bats
```

Comment:
This test is quite long.. Maybe let's put it in a sub-directory, so that "bats test/*.bats" doesn't run it.